notes-webapp/src/main/webapp/skin/less/notes/notes.less

```
@@ -60,18 +61,20 @@
     z-index: 1031;
     background: #eeeeee;
 
+    .notes-content-wrapper {
+      min-height: calc(~"100vh - 210px");
```

there is new way introduced to manage Mobile ViewPort bug ([see it here](https://medium.com/@alekswebnet/fix-mobile-100vh-bug-in-one-line-of-css-dynamic-viewport-units-in-action-102231e2ed56))

```suggestion
      min-height: ~"calc(var(--100vh, 100vh) - 210px)";
```